bindep.txt

```diff
@@ -22,7 +22,7 @@ podman            [platform:rpm]
 python3-devel     [platform:rpm !platform:rhel-7 !platform:centos-7]
 python3-libvirt   [platform:rpm]
 python3-lxml      [platform:rpm]
-PyYAML            [platform:rpm !platform:rhel-8 !platform:centos-8 !platform:rhel-9 !platform:centos-9 !platform:fedora]
+PyYAML            [platform:rpm !platform:rhel-8 !platform:centos-8 !platform:rhel-9 !platform:centos-9 !platform:rhel-10 !platform:centos-10 !platform:fedora]
```

Comment:
we are we continueign to do this if we are skipping all release

we should just remove this line

Reply:
I agree with @SeanMooney 
What is the purpose of the `PyYAML` dependency, if we don't use it on any platform in recent history?
CentOS 7.9-2009[[3]](https://en.wikipedia.org/wiki/CentOS#cite_note-CentOS_7_Announcement-3) (12 November 2020) happened 4 years ago